.gitlab-ci.yml

```diff
@@ -73,6 +73,7 @@ build_preview:
     - touch Makefile.config
     - make examples
     - sync_integration_descriptions
+    - ./local/bin/py/build/pull_rbac.py $(get_secret 'dd_api_key') $(get_secret 'dd-app-key')
```

Comment:
will this need to be added to `build_live` as well?

Reply:
Added this here, thanks!